### PR TITLE
Add some test cases

### PR DESCRIPTION
**Pull request by snnn**
**Description**: 

1. Add test cases for the schema registry in onnxruntime_graph
2. Use ASSERT_STATUS_OK when appropriate

**Motivation and Context**
- Why is this change required? What problem does it solve?

Not required. But it makes better.

- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

**Comment by snnn** [comment]
Surprising, the vgg19 model is failing, but I didn't change any implementation code.

File: onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc [hunk 1]
```diff
@@ -35,6 +35,9 @@ void TestReduceOp(const std::string& op,
   test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kCudaExecutionProvider, kTensorrtExecutionProvider});  //TensorRT: result differs
 }
 
+//TODO:investigate why it is so slow. It need 12 seconds on an Azure Standard F48s_v2 (48 vcpus, 96 GiB memory)
+// machine in RelWithDebInfo build mode, but only 2 seconds on my local dev machine(4 cores).
+#ifdef NDEBUG
 TEST(ReductionOpTest, ReductionVariationTest) {
```
Comment (skottmckay) [inline onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc]: I don't think we should disable this as doing so will mean it gets ignored and we never investigate why it's slow.

File: onnxruntime/test/ir/onnx_model_test.cc [hunk 2]
```diff
@@ -8,13 +8,22 @@
 #include "core/graph/model.h"
 #include "core/graph/op.h"
 #include "core/session/onnxruntime_c_api.h"
+#include "test/providers/provider_test_utils.h"  //For ASSERT_STATUS_OK
 #include "test/test_environment.h"
```
Comment (skottmckay) [inline onnxruntime/test/ir/onnx_model_test.cc]: Can we split the two macros that check the status into maybe test/util/include/asserts.h? It's a little weird to include a file from test/providers here.

Comment (snnn) [reply onnxruntime/test/ir/onnx_model_test.cc]: Right. We should do it.

**Pull request merged**